apps/gateway/src/routes/api/v1/projects/[projectId]/versions/[versionUuid]/documents/handlers/run.test.ts

```diff
@@ -18,7 +19,6 @@ import { apiKeys } from '@latitude-data/core/schema'
 import { mergeCommit } from '@latitude-data/core/services/commits/merge'
 import { parseSSEvent } from '$/common/parseSSEEvent'
 import app from '$/routes/app'
-import { eq } from 'drizzle-orm'
```

Comment:
This was making the test fail in my machine. Drizzle from an app fails. It has to be run from core